tests/i18n/test_extraction.py

```diff
@@ -687,6 +687,25 @@ def test_all_locales(self):
         self.assertTrue(os.path.exists(self.PO_FILE_KO))
         self.assertFalse(os.path.exists('locale/_do_not_pick/LC_MESSAGES/django.po'))
 
+    def test_locales_format_valid(self):
```

Comment:
Now that you're using `subTest` and we're testing locales individually, it no longer makes sense to put these test methods under the class `MultipleLocaleExtractionTests`. How about a new class `LocaleValidityTests` or `LocaleValidityExtractionTests`, which is more verbose but fits in with the surrounding classes.

Reply:
Yeah! I was thinking the same way. I will create a separate class for all the tests now.